python/console/console_settings.ui

```diff
@@ -466,116 +583,46 @@
-           </widget>
-          </item>
-          <item row="0" column="0">
-           <widget class="QCheckBox" name="enableObjectInspector">
-            <property name="text">
-             <string>Enable Object Inspector (switching between tabs may be slow)</string>
-            </property>
-            <property name="checked">
-             <bool>false</bool>
-            </property>
-           </widget>
-          </item>
-         </layout>
-        </widget>
-       </item>
-       <item row="7" column="0">
-        <spacer name="verticalSpacer_2">
-         <property name="orientation">
-          <enum>Qt::Vertical</enum>
-         </property>
-         <property name="sizeHint" stdset="0">
-          <size>
-           <width>20</width>
-           <height>40</height>
-          </size>
-         </property>
-        </spacer>
-       </item>
-       <item row="1" column="0">
-        <widget class="QgsCollapsibleGroupBox" name="groupBoxTyping">
+       <item row="2" column="0">
+        <widget class="QgsCollapsibleGroupBox" name="groupBoxContextHelp">
          <property name="title">
-          <string>Typing</string>
+          <string>Contextual Help (F1)</string>
          </property>
          <property name="collapsed" stdset="0">
           <bool>false</bool>
          </property>
          <property name="saveCollapsedState" stdset="0">
           <bool>true</bool>
          </property>
-         <layout class="QGridLayout" name="gridLayout_11">
+         <layout class="QFormLayout" name="formLayout">
           <property name="verticalSpacing">
            <number>2</number>
           </property>
           <item row="0" column="0">
-           <widget class="QCheckBox" name="autoCloseBracket">
+           <widget class="QLabel" name="label_3">
             <property name="text">
-             <string>Automatic parentheses insertion</string>
+             <string>Open in</string>
             </property>
            </widget>
           </item>
-          <item row="1" column="0">
-           <widget class="QCheckBox" name="autoSurround">
-            <property name="text">
-             <string>Automatically surround selection when typing quotes or brackets</string>
-            </property>
-           </widget>
-          </item>
-          <item row="2" column="0">
-           <widget class="QCheckBox" name="autoInsertImport">
-            <property name="text">
-             <string>Automatic insertion of the 'import' string on 'from xxx'</string>
-            </property>
+          <item row="0" column="1">
+           <widget class="QComboBox" name="contextHelpBrowser">
+            <item>
+             <property name="text">
+              <string>Embedded Webview (developer tools)</string>
+             </property>
+            </item>
+            <item>
+             <property name="text">
+              <string>Default system web browser</string>
+             </property>
+            </item>
```

Comment:
Can you populate via code instead, and use user data for each item? Eg this approach: https://github.com/qgis/QGIS/blob/master/src/gui/layout/qgslayoutscalebarwidget.cpp#L132

(It's more robust vs populating in the ui)